### PR TITLE
[RELEASE] 20170517

### PR DESCRIPTION
**Pull request by MiguelMoll**
## Pull Requests
  - closes #2185 Additional HTTP proxy configuration [@nzoschke]
  - closes #2196 Mysql Version [@mattmanning]
  - closes #2197 newline after error msg [@nzoschke]
  - closes #2199 add missing lambda tag permissions [@mattmanning]

## Milestone Release
- [x] Release branch
- [x] Pass CI
- [x] Code review
- [x] Merge into master
- [x] Close milestone
- [ ] Release master
- [ ] Record release number: 
- [ ] Pass CI
- [ ] Update demo
- [ ] Deploy demo/httpd
- [ ] Deploy demo/rails
- [ ] Update staging
- [ ] Deploy staging/site-staging
- [ ] Deploy staging/console-staging
- [ ] Verbose [release notes](https://github.com/convox/rack/releases)
- [ ] Documentation review
- [ ] Publish release
- [ ] Release CLI

### CODE REVIEW

**Comment by MiguelMoll** [comment]
[20170517](https://trello.com/c/1ftQcy0Y/2462-20170517)

File: cmd/convox/apps.go [hunk 1]
```diff
@@ -6,8 +6,9 @@ import (
 	"strings"
 	"time"
 
+	cli "gopkg.in/urfave/cli.v1"
+
 	"github.com/convox/rack/cmd/convox/stdcli"
-	"gopkg.in/urfave/cli.v1"
```
Comment (mattmanning) [inline cmd/convox/apps.go]: Is this ok?

Comment (MiguelMoll) [inline cmd/convox/apps.go]: Didn't notice that. And it also compiled fine because `cli` is the actual pkg name. Wonder why goimports did this.

**Comment by mattmanning** [comment]
Docs review:
- `--http-proxy` install option https://trello.com/c/vaWizO12
- `--version` for mysql resource https://trello.com/c/PGPb2mY9

**Pull request merged**